app/src/main/java/com/android/sample/ui/components/Textfields.kt

```diff
@@ -30,23 +30,37 @@ fun PasswordTextField(
     onPasswordChange: (String) -> Unit,
     isPasswordVisible: Boolean,
     onPasswordVisibilityChange: () -> Unit,
+    passwordError: String?
 ) {
-  OutlinedTextField(
-      value = password,
-      modifier = Modifier.fillMaxWidth(0.8f).testTag("PasswordTextField"),
-      onValueChange = onPasswordChange,
-      label = { Text("Password") },
-      visualTransformation =
-          if (isPasswordVisible) VisualTransformation.None else PasswordVisualTransformation(),
-      trailingIcon = {
-        IconButton(onClick = onPasswordVisibilityChange) {
-          Icon(
-              imageVector =
-                  if (isPasswordVisible) Icons.Filled.Visibility else Icons.Filled.VisibilityOff,
-              contentDescription = if (isPasswordVisible) "Hide password" else "Show password")
-        }
-      },
-  )
+  Column(modifier = Modifier.fillMaxWidth(0.8f)) {
```

Comment:
The addition of a column allows you to put the error part, not using a lazy column should not be bad in this case as there is only one field to show in this composable.